contracts/HubPool.sol

```diff
@@ -570,9 +575,12 @@ contract HubPool is HubPoolInterface, Testable, Lockable, MultiCaller, Ownable {
     }
 
     function _relayRootBundleToSpokePool(uint256 chainId) internal {
+        // todo: do we need to check if this is nonzero here and elsewhere.
         AdapterInterface adapter = crossChainContracts[chainId].adapter;
         adapter.relayMessage(
             crossChainContracts[chainId].spokePool, // target. This should be the spokePool on the L2.
+            // todo: import function interface and use it directly to
```

Comment:
I tried this as well and was not able to get it to behave the way I thought you could. I remember seeing something like `myInterface.myFunction.getABI()` but this did not work and I could not find an equivalent syntax. will look a bit more or otherwise we can just leave it like this without any issues.